LICENSE

```diff
@@ -1,21 +1,22 @@
-    MIT License
+MIT License
 
-    Copyright (c) Microsoft Corporation.
+Copyright (c) 2022-2023 Microsoft Corporation.
+Copyright (c) 2023-2024 IEA Energy Technology Systems Analysis Programme.
```

Comment:
Does the license really need copyright statements?

Reply:
Normally a license needs a copyright statement. Here is an article about it on Wikipedia: https://en.wikipedia.org/wiki/MIT_License

Reply:
Oh right, I didn't know that. Cool :)